README.md

```diff
@@ -1 +1,60 @@
-# to-do-list
\ No newline at end of file
+# To-Do List
+
+> This project is focused towards building a to-do list. Objective of this project is to understand 
+- Webpack to bundle JavaScript
+- ES6 Syntax
+- Writing modular JS
+
+![screenshot](./src/to-d0.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages:
+1. ES6/JavaScript
+2. HTML
+3. CSS
+- Technologies used:
+1. Git
+2. GitHub
+3. VS Code
+
+## Live Demo
+
+[Live Demo Link](https://isaka-lumato.github.io/to-do-list/dist/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the repo with `git clone https://github.com/isaka-lumato/to-do-list.git`
+
```

Comment:
- [ ] Kindly complete the steps needed to run your project on a local machine. It should include all npm scripts needed to run. This might be obvious to you but not to someone else 😄

Reply:
> * [ ]  Kindly complete the steps needed to run your project on a local machine. It should include all npm scripts needed to run. This might be obvious to you but not to someone else smile

As I said earlier, include npm scripts. This is a webpack project hence the user needs to start the app from script tags. Now picture a user who is not familiar with npm, they will have issues starting your project. Kindly add all the scripts needed, starting from installing packages to how to start a wepack server